src/time_zone_lookup_test.cc

```diff
@@ -813,6 +814,7 @@ TEST(BreakTime, TimePointResolution) {
   const time_zone utc = utc_time_zone();
   const auto t0 = chrono::system_clock::from_time_t(0);
 
+  // clang-format off
```

Comment:
Ditto not wanting to defeat clang-format in cases like these (but not wanting to reformat them yet).